pom.xml

```diff
@@ -371,7 +371,11 @@
                 <extensions>true</extensions>
                 <configuration>
                     <instructions>
-                        <i-Package>com.github.davidmoten.rtree.*</i-Package>
+                        <Export-Package>
+                            com.github.davidmoten.rtree;
+                            com.github.davidmoten.rtree.geometry;
+                            com.github.davidmoten.rtree.internal;
```

Comment:
Why would we export an internal package?

Reply:
And why aren't we exporting all public packages?

Reply:
1. Some classes have reference to internal classes, which should also be available to the users. Eg. Geometries refers to classes in rtree.geometry.internal.
2. The easiest way seems to export all public packages. This is the least invasive. I think we can use wildcard '*'.

Reply:
I looked at RxJava bundle and it looks like this: 
```
'Export-Package': '!io.reactivex.rxjava3.internal.*, io.reactivex.rxjava3.*'
```
I assume that !  means don't export the internal stuff. We should do the same.

Reply:
I think something like this should work: 
!com.github.davidmoten.rtree.internal.*;
!com.github.davidmoten.rtree.geometry.*;
com.github.davidmoten.rtree.*;

Also [guava](https://github.com/google/guava/blob/master/guava/pom.xml) does the same.